package.json

```diff
@@ -40,6 +45,6 @@
     "react": "*"
   },
   "stripes": {
-    "type": "components"
+    "type": ""
```

Comment:
Are you sure about this? I've talked about this setting with Matt in the past.

Reply:
Yup! touched base with Matt on this. He pointed me to https://issues.folio.org/browse/STCLI-107 which would be a fix coming soon. So, the _"type": "components"_ instructs the CLI to skip a platform build when running unit tests. As not all erm-components exactly "dumb" components and we  need the platform to run tests, we shouldn't be having the _"type": "components"_ in the package.

Reply:
Awesome! 👏